cpu/kinetis_common/include/cpu_conf_kinetis.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_kinetis_common
+ * @{
+ *
+ * @file
+ * @brief           CPU specific definitions common to all Kinetis CPUs
+ *
+ * @author          Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef CPU_CONF_KINETIS_H
+#define CPU_CONF_KINETIS_H
+
+#include "cpu_conf_common.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @name   ARM Cortex-M specific CPU configuration
+ * @{
+ */
+#define CPU_DEFAULT_IRQ_PRIO            (1U)
+/* Each interrupt priority setting is 8 bits wide, for both CM4 and CM0+, but
+ * they are laid out differently. CM0+ concatenates the settings into 32 bit
+ * words, CM4 and CM7 uses direct 8 bit access */
+#define CPU_IRQ_NUMOF                   (sizeof(NVIC->IP))
```

Comment:
I've tested this change on K22F, there's no error for setting priorities in the NVIC for interrupts which are not linked to anything internally, it only simplifies the configuration a lot.